packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -12,6 +13,32 @@ export interface IButton {
   focus: () => void;
 }
 
+export interface IButtonStyles {
```

Comment:
Some comments to help people realize which is the appropriate style to override for customization?